tests/cases/fourslash/convertFunctionToEs6Class11.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='fourslash.ts' />
+
+// @allowNonTsExtensions: true
+// @Filename: foo.js
+////function Foo() {
+////    this.a = 0;
+////}
+////Foo[1] = function () {
+////}
+
+verify.codeFix({
+    description: "Convert function to an ES2015 class",
+    newFileContent:
+`class Foo {
+    constructor() {
+        this.a = 0;
+    }
+    static 1() {

```

Comment:
TIL this is legal.

Reply:
https://jsfiddle.net/w2htsgpn/

Reply:
https://www.typescriptlang.org/play?#code/MYGwhgzhAEBiD29oG8BQ1oQC5iwS2GgEYAKAShQF9VKg